cea/technologies/sewage_heat_exchanger.py

```diff
@@ -150,8 +142,8 @@ def calc_sewageheat( mcp, tin, w_HEX, Vf, cp, h0, min_m, L_HEX, tmin, ATmin):
     city districts. Energy and Buildings.
     """
 
-    mcp_min = min_m * cp # minimum sewage heat capacity in [kW/K]
-    mcp_max = Vf * w_HEX * 0.20 * 1000 * cp   # 20 cm is the depth of the active water in contact with the HEX
+    mcp_min = min_m * (cp/1000) # minimum sewage heat capacity in [kW/K]
```

Comment:
...CAPITAL LETTERS + It is called in some scripts differently, let's keep consistency in the naming